src/runtime_src/driver/xclng/drm/xocl/devices.h

```diff
@@ -72,6 +72,15 @@ struct xocl_board_private {
          .vendor = ven, .device=dev, .subvendor = PCI_ANY_ID, \
          .subdevice = subsysid, .driver_data =          \
          (kernel_ulong_t) &XOCL_BOARD_##priv
+
+struct xocl_dsa_vbnv_map {
+	uint16_t		vendor;
+	uint16_t		device;
+	uint16_t		subdevice;
+	char			*vbnv;
+	struct xocl_board_private	*priv_data;
+};
+
```

Comment:
Presumably this struct should have a flag field also so that flags can be used to indicate specific features associated with this particular DSA?

Reply:
flags are already contained in board_private structure